nodebuilder/tests/p2p_test.go

```diff
@@ -91,7 +92,7 @@ Steps:
 func TestBootstrapNodesFromBridgeNode(t *testing.T) {
 	sw := swamp.NewSwamp(t)
 	cfg := nodebuilder.DefaultConfig(node.Bridge)
-	cfg.P2P.Bootstrapper = true
+	p2p.IsBootstrapper = true
```

Comment:
Let's remove ___all___ the `p2p.IsBootstrapper = true`. Previously when we set it through the config, it was affecting only the specific node, but with global var it makes everyone a bootstrapper. It's fine to remove it; even we had `cfg.P2P.Bootstrapper = true` before. Our tests should not rely on assumption that someone is a bootstrapper and someone is not. Even if it breaks them now, will will fix them